app/controllers/concerns/idv/document_capture_concern.rb

```diff
@@ -3,11 +3,7 @@ module DocumentCaptureConcern
     def override_document_capture_step_csp
       return if params[:step] != 'document_capture'
 
-      if FeatureManagement.rails_csp_tooling_enabled?
-        override_document_capture_step_csp_with_rails_csp_tooling
-      else
-        override_document_capture_step_csp_with_secure_headers
-      end
+      override_document_capture_step_csp_with_rails_csp_tooling
```

Comment:
can we inline this and remove the `override_document_capture_step_csp_with_secure_headers` method?